src/index.ts

```diff
@@ -1,14 +1,17 @@
 import yargs, { Arguments } from 'yargs';
 
 import { CleanupCommand } from './commands/CleanupCommand';
-import { DummyCommand } from './commands/DummyCommand';
 import { HelmCommand } from './commands/HelmCommand';
+import { PresetCommand } from './commands/Nuxeo/PresetCommand';
```

Comment:
I used a simple hierarchy:
- "standalone" command -> `commands` folder
- sub command -> `commands/<parent_cmd>/` folder.
- sub sub command -> `commands/<parent_parent_cmd>/<parent_cmd>/`

Here, you are adding `commands/Nuxeo/XXXCmd` as "standalone" command.
Don't know if it was on purpose for testing; but for now it seems weird to me.

And `preset` cmd makes sence be be a sub command of `nuxeo` as it's completely tied to our needs.
Preview... I'm mitigate, but if not under `nuxeo`, could be interesting to use as well a namespace with a main command entrypoint like `jenkins`.